onnx_tf/handlers/backend/tfidf_vectorizer.py

```diff
@@ -0,0 +1,123 @@
+import numpy as np
+import tensorflow as tf
+
+from onnx_tf.handlers.backend_handler import BackendHandler
+from onnx_tf.handlers.handler import onnx_op
+from onnx_tf.handlers.handler import tf_func
+
+
+@onnx_op("TfIdfVectorizer")
+class TfIdfVectorizer(BackendHandler):
+
+  @classmethod
+  def args_check(cls, node, **kwargs):
+    if "pool_int64s" in node.attrs and "pool_strings" in node.attrs:
+      raise ValueError(
+          "Cannot set the pool_int64s and pool_strings in an input at the same time."
+      )
+
+  @classmethod
+  def _prepare_ngrams(cls, x, n, skip):
+    # This method transform input into n-grams for a specific skip
+    # input x is a 1D tensor
+    # ex1: x=[1,2,3,4] n=1 skip=0 output=[[1],[2],[3],[4]]
+    # ex2: x=[1,2,3,4] n=2 skip=0 output=[[1,2],[2,3],[3,4]]
+    # ex3: x=[1,2,3,4] n=2 skip=1 output=[[1,3],[2,4]]
+    count = x.shape[0] - n + 1 - skip
+    mul = skip + 1
```

Comment:
Variable names could be more intuitive, can we have `count` -> `num_ngrams` and i don't quite get what this `mul` mean.